.github/workflows/publish-to-pypi.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         python-version: '3.10'
 
     - name: Install dependencies
-      run: python -m pip install setuptools wheel
+      run: python -m pip install build
```

Comment:
Oh, and just noting that we should double check that the wheel and source files on TestPyPI look ok once this PR is merged.